_sass/_imla-nama.scss

```diff
@@ -15,8 +15,15 @@ html {
     font-style: normal;
     font-weight: 400;
     src: url("type/NotoNastaliqUrdu-Regular.woff2") format("woff2"),
-         url("type/NotoNastaliqUrdu-Regular.woff") format("woff"),
-         url("type/NotoNastaliqUrdu-Regular.ttf") format("truetype");
+         url("type/NotoNastaliqUrdu-Regular.woff") format("woff");
+}
+
+@font-face {
+    font-family: NotoNastaliqUrduBold;
+    font-style: normal;
+    font-weight: 400;
+    src: url("type/NotoNastaliqUrdu-Bold.subset.woff2") format("woff2"),
+         url("type/NotoNastaliqUrdu-Bold.subset.woff") format("woff");
```

Comment:
Here, you have given a different `font-family` name for the bold weight and kept its `font-weight` value as `400` (which is the regular weight). This works, but has two disadvantages:

1. We will need to change the `font-family` whenever we need to use the bold weight. It's not that much of a hit for our case, but imagine if we have numerous weights and styles of a typeface to deal with, in which case it quickly becomes cumbersome to remember all those unique `font-family` names.
2. If, for some reason, our webfont fails to load, the bold weight will be lost in the fallback font.

Instead, in the `@font-face` declaration, we can keep the same `font-family` name (`NotoNastaliqUrdu` in our case), and specify the correct `font-weight` in there too:

```
@font-face {
    font-family: NotoNastaliqUrdu;
    font-style: normal;
    font-weight: 700; /* 700 = bold */
    src: url("type/NotoNastaliqUrdu-Bold.subset.woff2") format("woff2"),
         url("type/NotoNastaliqUrdu-Bold.subset.woff") format("woff");
}
```

Then, whenever we need to use the bold weight, we just use `font-weight: bold` or `font-weight: 700` as usual.